Util/getIMOSType.m

```diff
@@ -0,0 +1,40 @@
+function [mtype] = getIMOSType(imos_parameter)
+% function mtype = getIMOSType(imos_parameter)
+%
+% A wrapper to get the type of an IMOS parameter
```

Comment:
would be maybe clearer to say data type or something equivalent, to know we're talking about float, int ...

Reply:
done - ```imos_parameter_str```